src/driver/DriverFactory.ts

```diff
@@ -28,6 +29,8 @@ export class DriverFactory {
                 return new MysqlDriver(connection);
             case "sqlite":
                 return new SqliteDriver(connection);
+            case "cordova-sqlite":
+                return new CordovaSqliteDriver(connection);
```

Comment:
can you briefly explain what are differences between sqlite and Cordova drivers?

Reply:
There are three main differences in the cordova plugin:
* it has to be loaded differently (see `loadDependencies()` in the Driver class
* it does not support named parameters. Therefore I had to overwrite `insert()` and `parametrize()`
* sql-queries are executed with `databaseConnection.executeSql()` and not `databaseConnection.all()`

Other than that they work identical

Reply:
if it does not support named parameters then how you escape parameters?

Reply:
Also since both websql and cordova plugins have almost same codebase, maybe its good time to make a good abstraction for them? Are you able to do that?

Reply:
* parameters are escaped with a ? and have to be in the correct order to mapped correctly
* It looks like WebSQL, Cordova SQLite and SQLite could get an abstraction, right?

Reply:
understood. yes they all have only few differences since they all are based on sqlite

Reply:
Should I add that to this pull request or open a seperate one?

Reply:
as you wish. We can finish with this PR and then do it, it will be easier to review